docs/en/standalone.md

```diff
@@ -1,138 +1,98 @@
-# QuickStart (Standalone Version)
+# OpenMLDB QuickStart (Standalone Mode)
 
-## Deployment
-### Modify Config
+## 1. Deployment
+### 1.1 Configuration
 1. Please replace `127.0.0.1` with the server IP if you want to access remotely
 2. (Optional) Please change the port number if it is occupied by another service
 
 * conf/tablet.flags
-   ```
+   ```bash
    --endpoint=127.0.0.1:9921
    ```
 * conf/nameserver.flags
-   ```
+   ```bash
    --endpoint=127.0.0.1:6527
    # set the same value as the endpoint in conf/tablet.flags
    --tablet=127.0.0.1:9921
-   #--zk_cluster=127.0.0.1:7181
-   #--zk_root_path=/openmldb_cluste
    ```
 * conf/apiserver.flags
-   ```
+   ```bash
    --endpoint=127.0.0.1:8080
    # set the same value as the endpoint in conf/nameserver.flags
    --nameserver=127.0.0.1:6527
    ```
-### Start service
+### 1.2. Starting service
 ```bash
 sh bin/start-all.sh
 ```
 If you want to stop, execute the command as follows 
 ```bash
 sh bin/stop-all.sh
 ```
-## Usage
-### Start CLI
+## 2. Usage Guideline
+### 2.1. Starting CLI
 ```bash
 # The value of host and port is the endpoint in conf/nameserver.flags
 ./openmldb --host 127.0.0.1 --port 6527
 ```
-### Create DB
+### 2.2. Creating Database and Tables
 ```sql
 > CREATE DATABASE demo_db;
-```
-
-### Create Table
-```sql
 > USE demo_db;
-> CREATE TABLE demo_table1(c1 string, c2 int, c3 bigint, c4 float, c5 double, c6 timestamp, c7 date, index(ts=c6));
+> CREATE TABLE demo_table1(c1 string, c2 int, c3 bigint, c4 float, c5 double, c6 timestamp, c7 date, INDEX(key=c1, ts=c6));
 ```
-**Note**: Specify at least one index and set the `ts` column which is used for ORDERBY. The `ts` column is the key in `index` option and can be setted with `timestamp` or `bigint` column only. 
-### Import Data
-Only support csv file format (a sample csv file can be downloaded [here](../../demo/standalone/data/data.csv))
+The `INDEX` function used in `CREATE TABLE` accepts two important parameters `key` and `ts`. 
+
+- `key` specifies the index column. If it is not specified, OpenMLDB will use the first applicable column as the index column automatically. Necessary indexes will be built when deploying online SQL. 
+- `ts` specifies the ordered column, which is used by `ORDER BY`. Only the `timestamp` and `bigint` columns can be specified as `ts`. 
+- `ts` and `key` also determine whether certain SQL queries can be executed offline, please refer to [Introduction to OpenMLDB's Performance-Sensitive Mode](performance_sensitive_mode.md) for more details.
+
```

Comment:
TTL can be specified  in `INDEX` function.  The conf of TTL contains `ttl_type` and ttl value.  We can set ttl as follows:
INDEX(key=c1, ts=c6, ttl=100m, ttl_type=absolute)
INDEX(key=c1, ts=c6, ttl=10, ttl_type=latest)
INDEX(key=c1, ts=c6, ttl=(120min, 100), ttl_type=absandlat)
INDEX(key=c1, ts=c6, ttl=(120min, 100), ttl_type=absorlat)
The default ttl_type is `absolute`, default ttl is 0

Reply:
lets move those detail to a user guide doc, not in the quickstart